.travis.yml

```diff
@@ -45,3 +45,9 @@ jobs:
 cache:
   directories:
     - $GOPATH/pkg/mod
+notifications:
+  slack:
+    rooms:
+      secure: opr9EexG4WwF/o9ygOP4MdSMmaR2UBeBDReDJv8qF6eflL0Mh5OtN++vRttpfnt9/HnEaaiO7hfp4BptatyjaO2VOnIU5h4lPX2NthmS2V0tx5Xq/Oo8jvyJ3IOAzLCt8RT3etmlimk9NCJm/ut8AejpnKv0nZqIyK805by2cRAwdswNNXN2/cCNlQAENSphnlkWuyo6Bsdtj7CwtqVkX2il5UMdk37qd1LzX0hM3hl0HlEbxmFD536imgnDbs3+13PuU/UDo2ggSvgLKOevGW5VXYhaCaRra/jYdDh/Mn0wgMB5354lP5OY59WrZyvlujCURyF/vV3iN+7IrC/vtAKAHYZZLxLAXekex2Pe6RIj3AKUlrp5lLUR9JP4dtfukRzrPclsFkO/gIXs4NgvDj5opjl0g7fIphgsaXGgsrjgvunjWovOEUx+l1vRuSXJF/hNG0OQnRRTCy5OQpxXgYLKQb6JyGn2l0UXQG9fuq4aoQ2ePQn+UbjFiGDEfaECk7ia69JJspfix4LSuwoHa2fNxh6XaA4z+oUmV+gLGuaG5q0yMMERwqrLzuS59m5oaux41xKFeUc4fG6ait7jrm4NRb/2ug/lqae2jifEcEAr084b/PwtpVpAP2KLDj9BaDzhnFKRjkPaj6yU2p9WJqbePXtrkw5x2Zs78dzdwZs=
+    on_success: change
```

Comment:
maybe lets set this to always as well?

Reply:
I remember we discussed this the other day. We chose to send the notification only if the build succeed where it was broken before. We do this in order not to flood the channel